### ISSUE TITLE
Does FABind+ output some binding affinity or confidence score in the result?

### ISSUE THREAD

**Issue by zhq90** [QizhiPei/FABind#20]
Does FABind+ provide a binding affinity or confidence score in its output to evaluate ligand-protein interactions?

**Comment by KyGao** [comment]
FABind+ doesn't have a module that's directly trained to predict binding affinity. However, we do provide a pre-trained confidence score head (available in `confidence_model.bin`). You can refer to the inference script [here](https://github.com/QizhiPei/FABind/tree/main/FABind_plus#inference-on-custom-complexes-1) for details. During inference, you can control whether to sample multiple pockets/conformations using the `--use-clustering` and `--infer-dropout` parameters.
